doc/yay.8

```diff
@@ -1,5 +1,5 @@
 '\ t
-.TH "YAY" "8" "2018\-07\-01" "Yay v8.972+" "Yay Manual"
+.TH "YAY" "8" "2019\-10\-21" "Yay v9+" "Yay Manual"
```

Comment:
This should be the exact version number. Also I don't think this justifies three separate commits.

Reply:
I was going to squash it on merge, but I can squash it in branch.
I'll put `yay v9.4`